src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let summ = 0;
+  const price = 40;
+  const saleMin = 20;
+  const saleMax = 50;
+  const sum = price * days;
+  const maxDays = 7;
+  const minDays = 2;
+
+  summ = sum;
+
+  if (days < maxDays && days > minDays) {
+    summ = sum - saleMin;
+  } else if (days >= maxDays) {
+    summ = sum - saleMax;
+  }
+
+  return summ;
```

Comment:
too many variables it's also not very good ;)

```suggestion
  const sum = 40 * days;

// here it's obvious that 7 is number of days, cause you compare it with days variable
  if (days >= 7) {
    return sum - 50;
  }
  
  if (days >= 3) {
    return sum - 20;
  }

  return sum;
```

Reply:
Добрий день, не зрозумів що не так?